gensim/models/word2vec.py

```diff
@@ -1577,23 +1577,23 @@ def __init__(self, source, max_sentence_length=MAX_WORDS_IN_BATCH, limit=None):
         self.limit = limit
 
         if os.path.isfile(self.source):
-            logging.warning('single file read, better to use models.word2vec.LineSentence')
+            logger.warning('single file read, better to use models.word2vec.LineSentence')
```

Comment:
If the class API contract supports it, this is no warning (maybe debug, at most).

If it's outside the API contract, this is an error and we should raise an exception, not log a warning.

Reply:
Clarified this message a bit, made it debug.